components/hab/src/command/bldr/channel/destroy.rs

```diff
@@ -18,18 +18,18 @@ use common::ui::{Status, UI};
 use {PRODUCT, VERSION};
 use error::{Error, Result};
 
-pub fn start(ui: &mut UI, bldr_url: &str, token: &str, origin: &str, channel: &str) -> Result<()> {
+pub fn start(ui: &mut UI, bldr_url: &str, token: &str, origin: &str, key: &str) -> Result<()> {
     let depot_client = DepotClient::new(bldr_url, PRODUCT, VERSION, None).map_err(
         Error::DepotClient,
     )?;
 
-    ui.status(Status::Deleting, format!("channel {}.", channel))?;
+    ui.status(Status::Deleting, format!("secret key {}.", key))?;
 
     depot_client
-        .delete_channel(origin, channel, token)
+        .delete_origin_secret(origin, token, key)
         .map_err(Error::DepotClient)?;
 
-    ui.status(Status::Deleted, format!("channel {}.", channel))?;
+    ui.status(Status::Deleted, format!("secret key {}.", key))?;
```

Comment:
Hmm, I'm a little confused.  Why was `components/hab/src/command/bldr/channel/destroy.rs` changed to destroy origin secrets instead?  Wouldn't it be better to leave the channel stuff alone and put the destroying secrets stuff into its own file?

Reply:
I had the same thought!

Reply:
:whoa: that's definitely a mistake.